Containerfile

```diff
@@ -0,0 +1,55 @@
+# This file will contain the configuration for containerized esi-leap
+
+# Use python as the parent image for esi-leap
+
+FROM python:3
+
+# Use label descriptor
+
+LABEL description="ESI-LEAP container file"
+
+# Maintainer details
+
+MAINTAINER <tzumainn@redhat.com>
+
+# The working directory for esi-leap inside the container
+
+WORKDIR /esi-leap
+
+
+# Copy the files of esi leap inside the container
+# Syntax : COPY <src> <destination>
+
+COPY . .
+
+# Run command to install dependencies for esi-leap
+
+RUN pip install esi-leap
+
+RUN pip install mysql-connector
+
+# Please make sure to have esi-leap.conf file in same directory as containerfile 
+# or make sure to mention the right path of the file below.
+
+COPY esi-leap.conf /etc/esi-leap/esi-leap.conf
+
+RUN mkdir -p /var/log/esi-leap
+
+RUN touch /var/log/esi-leap/esi-leap-dbsync.log
+
+# Expose the default port for esi-leap-api
+# Use -p flag while running the image to map ports
+
+EXPOSE 7777
+
+#USER root
+
+
+# Connect to database and Start processes for esi-leap-api and esi-leap-manager
+
+
+#RUN esi-leap-dbsync create_schema
```

Comment:
I'm guessing this is commented out because the schema might already exist; however, that won't always be the case. Is it possible to have a script that creates the schema if it doesn't exist?

Reply:
Yes. I made that change.